quickwit/quickwit-storage/src/metrics.rs

```diff
@@ -88,45 +88,48 @@ pub struct CacheMetrics {
     pub component_name: String,
     pub in_cache_count: IntGauge,
     pub in_cache_num_bytes: IntGauge,
-    pub hits_num_items: IntCounter,
-    pub hits_num_bytes: IntCounter,
-    pub misses_num_items: IntCounter,
+    pub hits_num_items: IntCounterVec<0>,
```

Comment:
Is there a point to having a `IntCounterVec<0>` instead of a `IntCounter`?

Reply:
`new_counter` does not take labels as pram, added a new helper function to fix it